### PR TITLE
Planck rev6 drop

### PR DESCRIPTION
**Pull request by dijidiji**
- Added Vial support for the Drop Planck rev6 including VialRGB lighting
- Enabled QMK RGB matrix to allow for the aforementioned VialRGB to work

Presumably this works for all Planck variants but I'm unable to test anything other than the Drop rev6.

### CODE REVIEW

File: keyboards/planck/keymaps/vial/olkb-planck-rev6.1.json:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (xyzz) [inline keyboards/planck/keymaps/vial/olkb-planck-rev6.1.json:Zone.Identifier]: Shouldn't be in the PR

**Review by xyzz — CHANGES_REQUESTED** [review]
If you've only tested the keymap with `rev6_drop`, then the keymap should go under the `rev6_drop` folder.

File: keyboards/planck/keymaps/vial/olkb-planck-rev6.1.json:Zone.Identifier (hunk 1 above)
Comment (dijidiji) [reply keyboards/planck/keymaps/vial/olkb-planck-rev6.1.json:Zone.Identifier]: Oops, seems to have snuck in.

**Comment by dijidiji** [comment]
> If you've only tested the keymap with `rev6_drop`, then the keymap should go under the `rev6_drop` folder.

vial keymap folder moved under `rev6_drop`, to be honest I had no idea keymaps could be that specific but it compiles and functions as expected!

**Pull request merged**